### PR TITLE
chore: Minor refactor

### PR DESCRIPTION
**Pull request by jamesgeorge007**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [x] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case
Refactored the respective code block to a more concise form.

### Breaking Changes
N/A

### Additional Info
N/A

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @hiroppy @evilebottnawi

File: lib/util.js [hunk 1]
```diff
@@ -66,19 +64,21 @@ module.exports = {
...
 
+    const hostNameIsTheSame = localPrefix.hostname === urlObject.hostname;
+    
     // publicPath has the hostname that is not the same as request url's, should fail
     if (
-      localPrefix.hostname !== null &&
-      urlObject.hostname !== null &&
-      localPrefix.hostname !== urlObject.hostname
+      !localPrefix.hostname &&
+      !urlObject.hostname &&
```
Comment (evilebottnawi) [inline lib/util.js]: Breaking change, don't modify `!== null`

**Comment by jamesgeorge007** [comment]
@evilebottnawi why is the build failing :thinking:

**Comment by jamesgeorge007** [comment]
@evilebottnawi why is the build failing :thinking:

**Comment by evilebottnawi** [comment]
because something broken, please read CI message

**Comment by jamesgeorge007** [comment]
Does the change make sense to you?

**Comment by jamesgeorge007** [comment]
@evilebottnawi hope everything's fine now :thinking:

**Pull request merged**